asreview/json_logging.py

```diff
@@ -0,0 +1,238 @@
+import json
+from collections import OrderedDict
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+
+import asreview
+from asreview.settings import ASReviewSettings
+from asreview.analysis.statistics import _get_labeled_order
+from asreview.analysis.statistics import _get_last_proba_order
+
+
+class Dict_Logger(object):
```

Comment:
Name. 

Should this be an abstract base class?

Reply:
I don't think so, it can be useful to have as a logger that doesn't save anything.